libraries/AP_DDS/README.md

```diff
@@ -253,7 +253,28 @@ requester: making request: ardupilot_msgs.srv.ModeSwitch_Request(mode=4)
 response:
 ardupilot_msgs.srv.ModeSwitch_Response(status=True, curr_mode=4)
 ```
- 
+## Commanding using ROS 2 Topics
+The following topic can be used to control the vehicle.
+
+- `/ap/joy` (type `sensor_msgs/msg/Joy`): overrides a maximum of 8 RC channels,
+at least 4 axes must be sent. Use `NaN` to disable the override of a single channel.
+
```

Comment:
Should add a note on how priority is determined. We also have the potential to let RC have priority with a switch it looks like.
https://github.com/ArduPilot/ardupilot/issues/7045#issuecomment-841997340

Reply:
Added a comment to explain that each channel defaults to RC after 1s of inactivity.